### ISSUE TITLE
Breaks Resource creations after updating Nova to 3.3.2

### ISSUE THREAD

**Issue by saleh-mir** [spatie/nova-tags-field#86]
I just updated Nova to v3.3.2, republished all static files too. Now the "create" button doesn't work. It works if I comment `Tags::make('Tags'),` line inside my Resource. 

I also tested it on the update resource page and that works fine!
I checked the console, here's the thrown error being logged in the console after clicking on "create":
```
TypeError: "e.t0.response is undefined"
    e http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    k http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    _invoke http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    t http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    o http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    default http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    t http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    default http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    createResource http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    e http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    k http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    _invoke http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    t http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    o http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    default http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    t http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    default http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    submitViaCreateResource http://jesse-ai.test/vendor/nova/app.js?id=304aaa11dab68a87b9cc:1
    It http://jesse-ai.test/vendor/nova/vendor.js?id=f9725df909a5af18907e:1
    n http://jesse-ai.test/vendor/nova/vendor.js?id=f9725df909a5af18907e:1
    _wrapper http://jesse-ai.test/vendor/nova/vendor.js?id=f9725df909a5af18907e:1
vendor.js:1:1003419
    Xt http://jesse-ai.test/vendor/nova/vendor.js?id=f9725df909a5af18907e:1
    Ft http://jesse-ai.test/vendor/nova/vendor.js?id=f9725df909a5af18907e:1
    Ht http://jesse-ai.test/vendor/nova/vendor.js?id=f9725df909a5af18907e:1
    It http://jesse-ai.test/vendor/nova/vendor.js?id=f9725df909a5af18907e:1
```

**Comment by volkv** [comment]
Yes. Its broken now
![image](https://user-images.githubusercontent.com/14368228/78995515-30389680-7b4b-11ea-93d1-646abfea3aaa.png)

**Comment by freekmurze** [comment]
I don't have the time to look at this now, but I'd accept a PR that fixes this.

**Comment by saleh-mir** [comment]
@freekmurze I'm new to sending PRs to Laravel packages. I followed this [link](https://laravel-news.com/setup-your-local-environment-for-open-source-package-contributions) about how to contribute to Laravel packages. I installed the package and ran:
```
composer install
npm install
```
And now `npm run dev` works, but `composer test` gives me:
```
  [Symfony\Component\Console\Exception\CommandNotFoundException]  
  Command "test" is not defined.                                  
```

I modified `resources/js/components/Nova/DetailField.vue`, and executed `npm run dev` again, but my changes didn't take effect. 

Could you help me with this so I can take a look?

**Comment by tswonke** [comment]
We had the same problem and went back to Nova 3.2.1 for the moment.
Sorry, no time either to dig into this, I‘m afraid :-(

**Comment by volkv** [comment]
Seems got fixed with `Nova 3.4.0`

**Comment by killgt** [comment]
Can confirm, something broke our plugins too on the 3.3.*. Can be closed
